biorbd_optim/constraints.py

```diff
@@ -71,11 +71,17 @@ def add_constraints(ocp, nlp):
             elif elem[0] == Constraint.Type.PROPORTIONAL_Q:
                 Constraint.__proportional_variable(ocp, nlp, x, elem[2])
             elif elem[0] == Constraint.Type.PROPORTIONAL_CONTROL:
-                # TODO: Paul = Raise error if INSTANT.END is used
+                if elem[1] == Constraint.Instant.END:
+                    raise RuntimeError(
+                        "A proportional control does not make sense at the last node (Instant.END)"
```

Comment:
That is a bit harsh!

Reply:
Changed in "Instant.END is used even though there is no control u at last node"